src/core/server/integration_tests/http/versioned_router.test.mocks.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const captureErrorMock = jest.fn();
+
+jest.doMock('elastic-apm-node', () => {
```

Comment:
NIT do we really need to define a "versioned" specific mock? Seems identical to the `router.test.mocks.ts`.

Reply:
By convention, we use a dedicated mock file for each test file. Ihmo it makes sense, as it avoid breaking other suites when you add mocks for a specific test/file.

Reply:
That's a valid point, thanks for the explanation!